lib/matplotlib/patches.py

```diff
@@ -2968,6 +2968,13 @@ def __call__(self, path, mutation_size, linewidth,
             else:
                 return self.transmute(path, mutation_size, linewidth)
 
+    @_register_style(_style_list, name="-")
+    class Curve(_Base):
```

Comment:
Why the superclass change? Now it won't have the `beginarrow` properties.

Reply:
The original idea was that none of these attributes make sense for Curve anyways, and that this way I don't have to hide head_length/head_width by writing a separate `__init__` (I could have restored beginarrow/etc. as deprecated (or not deprecated) attributes), but OTOH I guess someone *could* have been doing `blah.get_arrowstyle().beginarrow = True` to switch between Curve and Arrow, so I decided to go back to the old design.